server/app/models.py

```diff
@@ -165,6 +165,48 @@ def get_photo_url_list(self):
     def get_member_service_list(self):
         return self.member_services.all().values_list('id', flat=True) or []
 
+    def init_prices(self):
```

Comment:
You may add a simple unit test, like just checking there exists a `priceconfig` after executing this method.